Utilities/ReleaseScripts/scripts/git-publish

```diff
@@ -257,7 +257,6 @@ CMSSW_CATEGORIES={"Operations": ["Configuration/Applications",
   "RecoMuon/TrackingTools", "RecoMuon/TransientTrackingRecHit",
   "RecoParticleFlow/Benchmark", "RecoParticleFlow/Configuration",
   "RecoParticleFlow/PFBlockProducer", "RecoParticleFlow/PFClusterProducer",
-  "RecoParticleFlow/PFClusterShapeProducer", "RecoParticleFlow/PFClusterTools",
```

Comment:
`RecoParticleFlow/PFClusterTools` should not have been deleted

Reply:
Thanks!